### PR TITLE
Feature/nxl 7289682407 campus weather sidebar

### PR DESCRIPTION
**Pull request by elusiveuphoria**
## Monday.com Item

Monday Item ID: 7289682407

## Description

Global Sidebar link on campus weather page displaying properly.

## How Has This Been Tested?

Tested Links. Nearly all work, but Active Warnings is attempting to link directly to new hazards page. This and perhaps a few others will likely need further discussion about possible implementation of new features; or at the very least these can serve as placeholders until the rest of the site is built.

## Screenshots/GIF (if appropriate):

PR screen

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by Panguino — CHANGES_REQUESTED** [review]
lets make the dots art for the group optional so we can remove them in the courses sidebar

**Comment by elusiveuphoria** [comment]
I supposed this is _one_ way to do it;

Commit ID: 78b6a74df161a84b9ff1b07652c8534a7782009d

File: src/components/elements/SidebarGroup/SidebarGroup.module.scss [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
+	border-bottom: none;
+
+	&::after {
+		content: '';
+		position: absolute;
+		top: calc(var(--left-border-dot-top) - 5px);
+		left: -4px;
+		width: 9px;
+		height: 9px;
+		background-color: grey;
```
Comment (Panguino) [inline src/components/elements/SidebarGroup/SidebarGroup.module.scss]: @elusiveuphoria we should always be using our css vars for colors, I am sure you can find these colors in our global theme file

**Review by Panguino — CHANGES_REQUESTED** [review]
@elusiveuphoria we should always be using our css vars for colors, I am sure you can find these colors in our global theme file

File: src/components/elements/SidebarGroup/SidebarGroup.module.scss (hunk 1 above)
Comment (elusiveuphoria) [reply src/components/elements/SidebarGroup/SidebarGroup.module.scss]: I'm behind on the replies... resolved 1b5df3e9716d664d8d2cba3b3040e99673595322 I think...

**Review by Panguino — APPROVED** [review]
Looks good

**Pull request merged**